x-pack/plugins/ml/public/ui_actions/open_lens_vis_in_ml_action.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { Embeddable } from '@kbn/lens-plugin/public';
+import { createAction } from '@kbn/ui-actions-plugin/public';
+import { MlCoreSetup } from '../plugin';
+
+export const CREATE_LENS_VIS_TO_ML_AD_JOB_ACTION = 'createMLADJobAction';
+
+export function createLensVisToADJobAction(getStartServices: MlCoreSetup['getStartServices']) {
+  return createAction<{ embeddable: Embeddable }>({
+    id: 'create-ml-ad-job-action',
+    type: CREATE_LENS_VIS_TO_ML_AD_JOB_ACTION,
+    getIconType(context): string {
+      return 'machineLearningApp';
+    },
+    getDisplayName: () =>
+      i18n.translate('xpack.ml.actions.createADJobFromLens', {
+        defaultMessage: 'Create anomaly detection job',
+      }),
+    async execute({ embeddable }) {
+      if (!embeddable) {
+        throw new Error('Not possible to execute an action without the embeddable context');
+      }
+
+      try {
+        // const [{ convertLensToADJob }, [, { share }]] = await Promise.all([
```

Comment:
Did you want to keep all this commented out code?

Reply:
removed in https://github.com/elastic/kibana/pull/129762/commits/1f07a1b7f54b2430aa0c33f1cfdcb81cea88acfd